examples/01_plotting/plot_demo_glass_brain_extensive.py

```diff
@@ -19,7 +19,7 @@
 
 # %%
 # Load the data
-# ------------------
```

Comment:
doing some minor touches on this example to trigger its build to see what the changes look like